app/views/mappings/_show_line.html.haml

```diff
@@ -19,30 +21,36 @@
     - ont_name =   target_concept.links['ontology'] || target_concept.id
     - ont_link =   ont_name
   - cls_link = raw(get_link_for_cls_ajax(target_concept.id, ont_name, '_top'))
-  - type = 'Internal'
+  - source = inline_svg_tag 'summary/homepage.svg', class: 'mappings-table-icon'
+  - source_tooltip = "Internal"
 - else
   - cls_label = get_label_for_external_cls(target_concept.links["self"])
   - cls_link = raw("<a href='#{target_concept.links["self"]}' target='_blank'>#{cls_label}</a>")
   - ont_name = target_concept.links['ontology']
   - ont_link = link_to ont_name, target_concept.links['ontology'], target: "_blank"
-  - type = 'External'
+  - source = inline_svg_tag 'icons/external-link.svg', class: 'mappings-table-icon', width: '20px', height: '20px'
+  - source_tooltip = "External"
   
 %tr.human{:id => map_id}
-  %td
+  %td.mappings-table-mapping-to
     = cls_link
-  %td
+  %td{style: "width: 30%"}
     = ont_link
+  - if relations
+    %td
+      - relations&.each do  |r|
+        = r
+        %br/
+
   %td
-    - relations&.each do  |r|
-      = r
-      %br/
-  %td
-    #{source}
-    - if !process.nil?
-      - if translation?(process["relation"])
-        %img{:src => asset_path('sifr/english_language_flag.png'), :style => "padding: 5px", :align => "right", :title => "Traduction"}
+    .d-flex.justify-content-center{'data-controller': 'tooltip', title: type_tooltip, style: 'font-weight: bold;'}
```

Comment:
I just flipped the sections type and source to keep source in the end of the table